code/modules/food&drinks/food/snacks.dm

```diff
@@ -151,7 +152,10 @@
 			return 1
 
 //Called when you finish tablecrafting a snack.
-/obj/item/weapon/reagent_containers/food/snacks/CheckParts()
+/obj/item/weapon/reagent_containers/food/snacks/CheckParts(list/content)
+	..()
+	create_reagents(volume)
```

Comment:
This was dont in the create_item proc after checking if the resulting shit a reagent container or something
I thought its needed